packages/fs-detectors/src/detectors/local-file-system-detector.ts

```diff
@@ -44,7 +44,7 @@ export class LocalFileSystemDetector extends DetectorFilesystem {
     };
     return dir.map(dirent => ({
       name: dirent.name,
-      path: path.join(dirPath, dirent.name),
```

Comment:
**question (non-blocking):** what bug(s) are being fixed with the `fs-detectors` changes?

Would it make sense to extract `fs-detectors` changes and deploy them before the project selector?